components/telegram_bot_api/actions/delete-message/delete-message.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable camelcase */
+const telegram_bot_api = require("../../telegram_bot_api.app.js");
+
+module.exports = {
+  key: "telegram_bot_api-delete-message",
+  name: "Delete a Message",
+  description: "Deletes a message",
```

Comment:
Can you add the api endpoint in the `description` prop. Same goes to the rest of action components